gtd-timer/Program.cs

```diff
@@ -12,6 +14,7 @@ public static void Main(string[] args)
 
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>();
```

Comment:
previous formatting was correct. this one is not

Reply:
Returned previous